packages/core/src/percy.js

```diff
@@ -423,12 +432,89 @@ export class Percy {
     return syncMode;
   }
 
+  // This specific error will be hard coded
+  async checkForNoSnapshotCommandError() {
```

Comment:
Why it needs to be hardcoded? 
Even if snapshot command is not called build is still created right?

Only in case of defer-uploads it will not happen, in that case we can hard code right?

Do we really need logs for this 🤔

Reply:
This suggestion is not related to the build failure. Moreover, this is related if the snapshot command was called or not, which means the user is aware of the snapshot command.
- We handed it here as to find the condition for this issue is quite simple, and we will be saving 1 network request as well. Another benefit was, that to detect the no snapshot command called, we need to have the info logs. So we have to send 16 - 20 log lines to API to process and provide this error, which can be found here.
- Mostly we have 3 cases for for snapshot upload
1. through script
2. through CLI ( npx percy snapshot <yaml>
3. though cli-upload
All this will be covered here.